README.md

```diff
@@ -65,6 +65,16 @@ def hello():
     return 'hello world'
 ```
 
+add key, value for your tracing record, like:
+
+```python
+
+zipkin.logging(id=1, user_id=2)
```

Comment:
I think this isn't what logging usually means. more typically it is tagging, right?